libcloudforensics/providers/gcp/forensics.py

```diff
@@ -227,3 +227,47 @@ def CreateDiskFromGCSImage(
       'md5Hash': md5_hash_hex
   }
   return result
+
+
+def InstanceNetworkQuarantine(project_id: str,
+                        instance_name: str,
+                        exempted_src_ips: Optional[List[str]] = None,
+                        enable_logging: bool = False) -> None:
+  """Put a Google Cloud instance in network quarantine.
+
```

Comment:
Can you be a bit more verbose about what network quarantine implies here? e.g. moving to another VPC network, applying firewall rules, etc.

Reply:
Done